app/head.mjs

```diff
@@ -1,15 +1,17 @@
-import { getStyles }  from '@enhance/arc-plugin-styles'
+import { getStyles } from '@enhance/arc-plugin-styles'
 
 const { linkTag } = getStyles
 
-export default function Head () {
+export default function Head(state) {
+  const { store = {} } = state
+  const { pageTitle = 'Enhance Starter Project' } = store
```

Comment:
This is totally debatable, but I'm wondering if setting the fallback here could divert folks who haven't read the preflight docs away from finding preflight in the first place. As in, they may just update the fallback string here without digging deeper? Will follow this up in the preflight diff below.

Reply:
Interesting. We could move setting the default title to the preflight map. It is kind of "6 on one hand" but your point about nudging people to use the pattern is a good one. I will make this change.